ConwaysGameOfLife.Tests/GameControllerShould.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using Xunit;
+
+namespace ConwaysGameOfLife.Tests
+{
+    public class GameControllerShould
+    {
+        StubOutput _output;
+
+        StubInput _input;
+
+        public GameControllerShould()
+        {
+            _output = new StubOutput();
+            _input = new StubInput();
+        }
+        [Fact]
+        public void FormatUniverse_GivenUniverseOfAllDeadCells() 
+        {
+            var universe = new Universe(3, 3);
+            var controller = new GameController(universe, _output, _input);
+            var expected = "💀💀💀\n💀💀💀\n💀💀💀\n";
+
+            controller.DisplayUniverse();
+
+            Assert.Equal(expected, _output.GetWriteLine());
+        }
+
+        [Theory]
+        [InlineData("3,3", 3, 3)]
+        [InlineData("10,10", 10, 10)]
+        public void ReturnUniverseString_GivenValidInput(string input, int gridWidth, int gridLength)
+        {
+            var universe = new Universe(gridWidth, gridLength);
+            var controller = new GameController(universe, _output, _input);
+            _input.GetReadLine(input);
+            var expected = input;
+
+            var result = controller.CreateValidUniverseString();
+
+            Assert.Equal(expected, result);
+        }
+
+        //TODO: how to test error message in CreateValidUniverseString

```

Comment:
Probably want to have a separate test for this and preload the `StubInput` with different inputs so that when your `CreateValidUniverseString` function is called in each test, the inputs are whatever you want them to be.